RusimbiPatrick/Banka
# #165310841 built user login endpoint

**What does this PR do?**
It creates a user login endpoint.

**Description of Task to be done?**
1. It adds a user login controller in the user controller file.
2. It validates information submitted during login such as email and password.
3. It adds a route for user login.
4. It adds a service for user login.

**How should this be manually tested?**
Start the Server and POSTMAN.
Browse the following endpoint route: localhost:3000/api/v1/auth/login.
Make sure that you have an account or create one in advance.
Submit the required information - email and password - with POST method.
Check if the validations work once you don't submit any of the required information.
After passing the validations, you should receive a 200 HTTP Status Code along with the information related to your account such as firstname, lastname, and email.
